packages/hoc-withtooltip/src/withTooltip.js

```diff
@@ -4,6 +4,7 @@ import wrapDisplayName from "recompose/wrapDisplayName";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 import values from "object.values";
+import ReactCSSTransitionGroup from "react-addons-css-transition-group";
```

Comment:
This is an outdated version. Use this one: https://reactcommunity.org/react-transition-group/

Also, it should be in the package's dependencies list. Go to the package folder and use `yarn add react-transition-group` there.

Reply:
We also should remove the animations set on tooltip component:
https://github.com/CraveFood/farmblocks/blob/dabb0163321f0cb7af087835f9d591cc2f2db9d9/packages/tooltip/src/styledComponents/Tooltip.js#L80